irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/relationships/SingleLevelBomAsBuilt.java

```diff
@@ -34,19 +34,18 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.extern.jackson.Jacksonized;
 import org.eclipse.tractusx.irs.component.GlobalAssetIdentification;
 import org.eclipse.tractusx.irs.component.LinkedItem;
 import org.eclipse.tractusx.irs.component.MeasurementUnit;
 import org.eclipse.tractusx.irs.component.Relationship;
 import org.eclipse.tractusx.irs.component.enums.AspectType;
 import org.eclipse.tractusx.irs.component.enums.BomLifecycle;
+import org.eclipse.tractusx.irs.edc.client.RelationshipSubmodel;
 
 /**
  * SingleLevelBomAsBuilt
  */
 @Data
-@Jacksonized
```

Comment:
why removed?

Reply:
`@Jacksonized` does nothing without the `@Builder` annotation. It produced a warning so I removed it. But since we have the builder annotation for the other relationship models, I guess it makes more sense to add the builder instead of removing jacksonized